ramp-engine/ramp_engine/dispatcher.py

```diff
@@ -253,6 +259,10 @@ def update_database_results(self, session):
             update_all_user_leaderboards(session,
                                          self._ramp_config['event_name'])
 
+    @staticmethod
+    def _reset_single_submission(session, submission_id):
+        set_submission_state(session, submission_id, 'new')
```

Comment:
Maybe just directly call this set_submission_state in the dispatcher loop, without the added indirection of the added static function?